### DISCUSSION TITLE
Prochaines fonctionnalités

### DISCUSSION THREAD

**Discussion by eric-taix** [eric-taix/volley34-mobile#302] [Ideas]
Le déploiement de la version 1.1.0 est en cours donc il est temps de discuter de la prochaine versions et des  grosses fonctionnalités à embarquer. Ci-après les idées qui me viennent d'après ce que l'on a déjà plus ou moins discuté / échangé (sans ordre de priorité). Je vous invité par ailleurs à ajouter des sujets si ils sont manquants.

Note: Je vais créer une discussion par sujet ce qui permettra de préciser les détails fonctionnelles et techniques pour chaque idée. Cette discussion ci est plus pour lister les sujets possibles et échanger sur leur priorité.

1- Suppression des feuilles de matchs et validation du score par les 2 équipes : #303 
2- Validation d'un report par les 2 équipes (à priori même chose que la validation de #303)
3- Pouvoir échanger des messages avec une équipe adverse sans connaitre les numéros de téléphone
4- Mettre en place un scorer (en option avec statistiques de l'équipe : points + fautes au service / réception / block / attaque /...)

**Comment by matt-aubert** [comment]
ok, quelques idées à la volée :
- Pusher des notifications comme, les matchs à venir, tournois, rappel de saisie de score ...
- permettre extract calendrier équipe (voire club, je crois que cette issue fait déjà parti du backlog)
- intégrer une interface de coaching stratégie : terrain de volley 2D avec possibilité de positionner les joueurs, dessiner etc. pour le coaching
- intégrer les règles avec outil de recherche (déjà dans le backlog il me semble)
- pouvoir partager certains écrans, si une équipe, un club a bien performé (ou juste pour clôturer la saison), pouvoir partager l'essentiel sur les réseaux FB etc.
- une page pour la comm ? par volley34 ou diffuser les activités récentes de la poule ? 
- intégrer un "sifflet" lors de la prise de points
